example/test_problem/Hydro/CCSN/Input__TestProb.CoreCollapse

```diff
@@ -20,7 +20,15 @@ CCSN_Eint_Mode   1                                 # Mode of obtaining internal
                                                    # ( 1=Temp Mode: Eint(dens, temp, [Ye])
                                                    #   2=Pres Mode: Eint(dens, pres, [Ye]) )
 
+CCSN_MaxRefine_FLAG1   1                           # enable limiting maximum refinement level 1 [0]
+CCSN_MaxRefine_FLAG2   1                           # enable limiting maximum refinement level 2 [0] ( CCSN_MaxRefine_FLAG1 should be turn on first)
+CCSN_MaxRefine_LV1     7                           # reduced maximum refinement level 1 to this value [MAX_LEVEL-2]
+CCSN_MaxRefine_LV2     7                           # reduced maximum refinement level 2 to this value [MAX_LEVEL-1]
+CCSN_MaxRefine_Dens1   1.0e11                      # central density threshold that reduces the maximum refinement level 1 [1.0e11]
+CCSN_MaxRefine_Dens2   1.0e12                      # central density threshold that reduces the maximum refinement level 2 [1.0e12]
```

Comment:
- I have changed the parameters' names.
- Now it supports that feature and I also corrected in `Input_Testprob`
- I replaced them.
- Now it can either use Flag1 or 2.